tools/stats/print_test_stats.py

```diff
@@ -41,12 +41,7 @@
     Version2Report,
     ReportMetaMeta,
 )
-from tools.stats.scribe import (
```

Comment:
Should we cleanup these methods as well if RDS is not used anywhere?

Reply:
Ah probably--will do in separate PR

Reply:
Oh, nah, realized we still use it in some other upload files. Will clean up separately + modify PR title to say it's only for print_test_stats